### DISCUSSION TITLE
My light is recognized as dimmable bulb instead of the configured light

### DISCUSSION THREAD

**Discussion by Raphcmx** [fsaris/EspHome-AwoX-BLE-mesh-hub#47] [Q&A] [answered]
device_info:
    # Example device type, see device info in HomeAssistant or MQTT message to find the 'product_id'
    product_id: 0x7E
    device_type: RGB
    name: EGLOPanel 10X120
    model: EPanel_120_10
    manufacturer: EGLO

even though the fitting device configuration is used the device gets recognized as "unknown" so it functions as a dimmable light

**Comment by fsaris** [comment]
Can you post some debug logging and your yaml config?

**Comment by Raphcmx** [comment]
Yes of course I forgot. I'm new to home assistant so i dont know if this is enough I found these warnings in the logger. Or do you need the whole installation process log? For better understanding, it recognizes the light just fine but as a dimmable so it is not possible to set the color of the light in any form. Also took out the secrets in the txt file in my configuration there are the right secrets.

[yaml.txt](https://github.com/fsaris/EspHome-AwoX-BLE-mesh-hub/files/13780204/yaml.txt)

[logger.txt](https://github.com/fsaris/EspHome-AwoX-BLE-mesh-hub/files/13780226/logger.txt)

**Comment by fsaris** [comment]
The logs look like HA logs. I'm interested in the logs from esphome

**Comment by Raphcmx** [comment]
What settings do I need to change so I can see older logs?

Now I am only able to see this: 

INFO ESPHome 2023.12.5
INFO Reading configuration /config/esphome/bluetooth-proxy-buro.yaml...
INFO Updating https://github.com/fsaris/EspHome-AwoX-BLE-mesh-hub@None
WARNING GPIO2 is a strapping PIN and should only be used for I/O with care.
Attaching external pullup/down resistors to strapping pins can cause unexpected failures.
See https://esphome.io/guides/faq.html#why-am-i-getting-a-warning-about-strapping-pins
INFO Starting log output from /dev/ttyUSB1 with baud rate 115200
[22:42:31][D][mesh_device:353]: online status report: mesh: 13379, on: 1, color_mode: 0, transition_mode: 0, w_b: 127, temp: 102, c_b: 100, rgb: 3EDFCF 
[22:42:31][I][mesh_device:418]: 13379: ON temp: 102 (127 %) ONLINE

I think thats not what you need :D

**Comment by fsaris** [comment]
Older logs are not kept by esphome.

I'm interested in the logs until it connects to your light and reports the device info over mqtt.

Open the log view and restart the esp device. Wait until it only logs the `online status report`. That should give enough info.

Did you reinstall after updating the yaml?

You should disable the `status_led` in your yaml because the used pin doesn't match your board.

**Reply by Raphcmx** [reply]
[logs_bluetooth-proxy-buro_run.txt](https://github.com/fsaris/EspHome-AwoX-BLE-mesh-hub/files/13801482/logs_bluetooth-proxy-buro_run.txt)

That should be the right file. This is the whole installation log when I install the yaml on the device hope this helps.

Yes I reinstalled several times but it is the same outcome everytime

What is the command for disabling the status_led? 

Thanks for you patience :D

**Reply by fsaris** [reply]
> What is the command for disabling the status_led?

Comment out (or remove) this part of the `yaml` 
```
status_led:
  # Match this with the board your using
  pin: GPIO2
```

or use `GPIO16` as that's should be the led pin for a `nodemcu-32s`

I see 2 different devices reported in the logs. 

```
MAC report, dev [21052]: productID: 85 mac: A4:C1:38:56:D2...
MAC report, dev [13379]: productID: 7E mac: A4:C1:38:BE:B4...
```

So I expect you configured the wrong device info:

```
-  product_id: 0x7E
   device_type: RGB
   name: EGLOPanel 10X120
   model: EPanel_120_10
   manufacturer: EGLO
```

Not sure about the other device. It's also a panel? The just add the same config and replace `product_id` with `0x85`

**Reply by Raphcmx** [reply]
The one device is an Eglo Plug Plus the wifi outlet, do I need to configure both devices? I am only interested in de Eglo Panel

**Reply by Raphcmx** [reply]
[config.txt](https://github.com/fsaris/EspHome-AwoX-BLE-mesh-hub/files/13801787/config.txt)
[logs_bluetooth-proxy-buro_run.txt](https://github.com/fsaris/EspHome-AwoX-BLE-mesh-hub/files/13801788/logs_bluetooth-proxy-buro_run.txt)

This is the new config and log. I changed the things you said, installed it and then restarted the whole system but it is still recognized as a dimmable light. Do I need to reconfigure mqtt or do I need to delete the devices beforhand?

**Reply by fsaris** [reply, marked as answer]
no HA should automatically update the device when a new device setup is received. To be sure check if the Logs of HA don't mention something about this failing.

I updated the main version to log what device info is matched/found in the logs of the ESP. Could you try the updated version?

Your plug can be configure using. I't not required to have it configured, but then you can also control is using HA for instance :)

```
-  product_id: 0x85
   device_type: PLUG
   name: EGLO PLUG PLUS
   model: SMPWBm10GEa
   manufacturer: EGLO
   icon: mdi:power-socket-de
```

**Reply by fsaris** [reply]
check your HA for logs like:

```
Platform mqtt does not generate unique IDs. ID awox-A4:C1:38:5A:CC:88-light already exists - ignoring light.....
```

Then the light features indeed don't automatically update in HA

**Reply by Raphcmx** [reply]
Thanks for your help I configured the Plug Plus and now it is working fine after reinstalling mqtt and the esp :). Now I am curious, is there a way to use the animations which are implemented in the awox app somehow? I read some things about the light service it should be possible with this now right?

**Reply by fsaris** [reply]
Nice that it works now.

The animations are currently not implemented.
